packages/utils/src/severity.ts

```diff
@@ -1,20 +1,28 @@
-import { Severity } from '@sentry/types';
+import { Severity, SeverityLevel } from '@sentry/types';
 
-import { SeverityLevel, SeverityLevels } from './enums';
+// TODO: Should `severityFromString` be deprecated?
+
+// Note: Ideally the `SeverityLevel` type would be derived from `validSeverityLevels`, but that would mean either
+//
+// a) moving `validSeverityLevels` to `@sentry/types`,
+// b) moving the`SeverityLevel` type here, or
+// c) importing `validSeverityLevels` from here into `@sentry/types`.
+//
+// Option A would make `@sentry/types` a runtime dependency of `@sentry/utils` (not good), and options B and C would
+// create a circular dependency between `@sentry/types` and `@sentry/utils` (also not good). So a TODO accompanying the
+// type, reminding anyone who changes it to change this list also, will have to do.
+
+export const validSeverityLevels = ['fatal', 'error', 'warning', 'log', 'info', 'debug', 'critical'];
```

Comment:
If we export this it leaks into the API. Do we want this just for a test?

Reply:
Totally with you. This is a perfect example of why I want to go to selective exports in the utils package.

For the moment, I'm going to leave this be, until we either do that or decide we're not doing that.